src/CMakeLists.txt

```diff
@@ -97,9 +97,20 @@ if(NOT TODBC_WINDOWS)
   set_source_files_properties(odbc.c PROPERTIES COMPILE_FLAGS -fvisibility=default)
 endif()
 
-set_target_properties(taos_odbc PROPERTIES
-    VERSION 0.1)
-install(TARGETS taos_odbc)
+set_target_properties(taos_odbc PROPERTIES VERSION 0.1)
 
-add_subdirectory(tests)
+if(TODBC_WINDOWS)
+  if(TODBC_X86)
+    set(INSTALL_PREFIX "C:/TDengine/taos_odbc/x86")
```

Comment:
INSTALL_PREFIX is normally provided for cmake tool user to specify, such as
cmake ... -DINSTALL_PREFIX="...."